shopping-cart/shopping-cart-java/pom.xml

```diff
@@ -221,8 +221,8 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <lagom.version>1.6.0-RC2</lagom.version>
-        <akka.version>2.6.0-RC2</akka.version>
+        <lagom.version>1.6.0-RC3</lagom.version>
+        <akka.version>2.6.1</akka.version>
```

Comment:
IIUC, Lagom `1.6.0-RC3` already uses Akka `2.6.1`. Do we need the explicit version of Akka?

Hmm, this property may be a leftover from the times when `akka-discovery` was not [pulled transitively](https://github.com/lagom/lagom-samples/blob/1.5.x/shopping-cart/shopping-cart-java/pom.xml#L192). I'd expect [any akka artifact](https://github.com/lagom/lagom-samples/blob/1259224e7e2cfb4d435584232456e6ff10610a34/shopping-cart/shopping-cart-java/pom.xml#L189-L198) in this sample to be already listed on the Lagom's BOM so there'd be no need to add them explicitly. 

I guess we can do a dependency cleanup on a separate PR.